src/Raven.Client/Json/Converters/BlittableJsonConverter.cs

```diff
@@ -9,32 +10,30 @@ namespace Raven.Client.Json.Converters
 
     internal sealed class BlittableJsonConverter : RavenJsonConverter
     {
-        //Todo To consider put context as part of the converter and not part of the reader
-        //public BlittableJsonConverter(JsonOperationContext context)
-        //{
-        //    _context = context;
-        //}
-
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
         {
             writer.WriteValue(value);
         }
 
         public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
         {
-            //Todo To consider put context as part of the converter and not part of the reader
-            //if (reader.Value is BlittableJsonReaderObject blittable)
-            //{
-            //    return blittable.Clone(_context);
-            //}
-            //throw new Exception($"Value must be {nameof(BlittableJsonReaderObject)}");
+            var blittableReader = reader as BlittableJsonReader;
+            if (blittableReader == null)
```

Comment:
Can use `reader is BlittableJsonReader blittableReader`

Reply:
done